terraform/environment/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.84.0"
+    }
+  }
+
+  backend "azurerm" {
+  }
+}
+
+provider "azurerm" {
+  features {}
+  #skip_provider_registration = true
```

Comment:
This was needed in the past when a user does not have enough access permissions to the resource group. Should we delete it?

Reply:
I think so.

Reply:
Issues with provider registration is something first-time users will face. It relates to:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers#registration

Reply:
No service principal creation is required in the ELMS files so we will remove the provider registration.